template.yaml

```diff
@@ -6,6 +6,11 @@ Globals:
   Function:
     Timeout: 30
 
+Parameters:
```

Comment:
not sure why this is a param instead of a constant

Reply:
Because this is an external value not under the control of the lambda developer (me) and making this configurable also means that we can change to use staging, for instance, if needed.